cli/command/image/build_linux_test.go

```diff
@@ -15,10 +16,12 @@ import (
 	"github.com/gotestyourself/gotestyourself/assert"
 	is "github.com/gotestyourself/gotestyourself/assert/cmp"
 	"github.com/gotestyourself/gotestyourself/fs"
+	"github.com/gotestyourself/gotestyourself/skip"
 	"golang.org/x/net/context"
 )
 
 func TestRunBuildResetsUidAndGidInContext(t *testing.T) {
+	skip.If(t, os.Getuid() != 0, "require root to run")
```

Comment:
It looks like this only needs root because of `archive.Untar`. I don't think this test actually needs to use that.

Maybe that will allow us to move it back to `build_test.go` ?